test/spectral/tests/enabled-rules.test.js

```diff
@@ -317,6 +323,48 @@ describe('spectral - test enabled rules - OAS3', function() {
       '`number_of_connectors` property should be equal to one of the allowed values: 1, 2, `a_string`, 8'
     );
   });
+
+  it('test major-version-in-path rule using mockFiles/oas3/enabled-rules.yml', function() {
+    expect(allOutput).toContain(
+      'Major version segment not present in either server URLs or paths'
+    );
+  });
+
+  it('test major-version-in-path using mockFiles/oas3/multiple-major-versions.yml', async () => {
```

Comment:
We've been putting tests for our custom rules in a separate directory in `test/spectral/tests/custom-rules`.

The tests above use the output from running against `enabled-rules.yml`, so they fit well in this file.

Since this test and the test below, however, invoke the validator with new api definitions, they may be better off in the `test/spectral/tests/custom-rules` directory.

Reply:
Fixed.